Packs/EWS/Integrations/EwsExtension/EwsExtension.yml

```
@@ -188,7 +188,7 @@ script:
         The maximum number of entries per page.
         Can be an integer between 1 and 5000. The default value is 100.
```

the server will add it automatically
```suggestion
        Can be an integer between 1 and 5000.
```